src/main/java/com/stockexchange/domain/order/entity/OrderTable.java

```diff
@@ -0,0 +1,57 @@
+package com.stockexchange.domain.order.entity;
+
+import com.stockexchange.domain.stock.entity.Stock;
+import com.stockexchange.domain.user.entity.User;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@NoArgsConstructor
+@Table(name = "order_table")
+public class OrderTable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_id")
+    private Long orderId;
+
+    @Column(name = "order_count")
+    private Integer orderCount;
+
+    @Column(name = "order_price")
+    private Integer orderPrice;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "order_type")
+    private OrderType orderType;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "order_status")
+    private OrderStatus orderStatus;
+
+    @Column(name = "order_remain_count")
+    private Integer orderRemainCount;
```

Comment:
```suggestion
    private int orderRemainCount;
```

기본형 으로 사용 가능한지 확인해주세요